### DISCUSSION TITLE
Where is Time To Die (TTD) option?

### DISCUSSION THREAD

**Discussion by LEOMODE** [Hekili/hekili#1078] [Q&A]
I was told that TTD could help a little better in terms of what skill I can use instead of just assuming as if the mob is still there. I couldn't find anywhere in the option. Is this just built-in and calculated automatically?

Also, I noticed that when I played a ranged dps (hunter), instead of toggling Automatic or AOE mode, I just first shoot multi-shot to activate AOE mode while I'm at Automatic mode, because once it detects more than 3 enemies it will do AOE mode anyway. Is this recommended or should I still toggle (by turning on AOE mode, then once the add-on actually detects AOE then switch Automatic Mode) for the best damage output?

Lastly, would you be able to add a healer dps option also? (disc priest for instance)

Thanks for all your work I really respect this add-on!

**Comment by Hekili** [comment]
Time to die is automatically calculated and factored into decision-making.  Some priorities make better use of it than others.

Yes, Automatic + Damage-based Target Detection works the way you described.  Using dual displays or toggling can be important when you want to focus down a single target despite there being multiple targets, etc.  Whatever works for you is the right choice.

I *can* add healer DPS modules, but they require almost as much work as a regular DPS module, and there's less benefit.  So they're lower priority.  If I have time and no other pressing issues, I might add them at some point.  They're not an urgent priority, though.

**Reply by LEOMODE** [reply]
Thanks. One important and follow-up question which I forgot to ask:

For Shadow Priest, when I Mind Flay, I am casting that spell for quite some time. I do see new skills popping up as my next skill to cast. Does that mean if I'm casting Mind Flay or Mind Sear, do I need to execute my next move once the entire casting on Mind Flay or Mind Sear is over, or do I need to cast it as soon as possible?

Another example is when I'm casting Mind Sear, I need to cast Searing Nightmare. In that case I know that I have to cast it during I'm casting Mind Sear, but if I'm casting Mind Flay or Mind Sear and I see Devouring Plague or Mind Blast, does it mean I need to cast it during or after Mind Flay/Mind Sear?

Thanks in advance.

**Reply by Hekili** [reply]
Assuming you have the default "delays" showing, if it says to do something now, do it now. If it shows that you're waiting (CD spiral, timer, etc.) then wait.

**Reply by LEOMODE** [reply]
Thank you but I'll be greatly appreciated if you can elaborate more on how to know when Hekili is saying 'do something now' or 'wait'. I wish there is more tutorial on this :)

What I can speculate is that when the icon is spiraling (e.g. as well as showing the countdown text) that's when I have to wait until it becomes 0 (Devouring Plague). But I see no spiraling or countdown text on Mind Blast. 

Would I be correct? Thanks.

**Reply by Hekili** [reply]
Yes, if it looks like a regular icon, no spiral, ready to go -- that's now.

If there's a spiral (or if you've turned on a text timer), wait for the spiral to go away.

In this regard, the addon uses the same visual metaphor for timing as the rest of the game.

**Reply by LEOMODE** [reply]
Appreciate your response. I tested various add-ons and yours is the best. So keep this up!
